packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -156,6 +156,34 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
         />
       );
       break;
+    case roomNotFoundPageTrampoline():
+      pageElement = (
+        <>
+          {
+            /* @conditional-compile-remove(rooms) */ <NoticePage
```

Comment:
Soft suggestion:
Instead of conditionally compiling these JSX, add a single trampoline that returns all the strings necessary for beta even on stable build, by providing empty default values.

There are examples of this strategy in the codebase.